### ISSUE TITLE
Scheduled departure command issue

### ISSUE THREAD

**Issue by mikeios** [jonasman/TeslaSwift#118]
Hi,

Thank you very much for your library
I try to send scheduledDeparture command to the vehicle. However, I cannot initialize the ScheduledDepartureCommandOptions because I get the following error message: 'ScheduledDepartureCommandOptions' initializer is inaccessible due to 'internal' protection level

Did anyone already get this issue? Is there a way to solve this problem?

I'm working on an app based on SwiftUI

Thank you very much

**Comment by jonasman** [comment]
we need to make the init public

**Comment by mikeios** [comment]
Thank you for your answer. Is it something i can do myself? i did not find any way to do this.

**Comment by jonasman** [comment]
yeah you can make a PR or wait for me to do it

**Comment by mikeios** [comment]
Yes thank you.
What I did is that I installed the package manually instead of using Swift Package Dependencies Manager. This way I was able to edit the file and make the init public, that's why I initially closed the issue.
I have to admit that I'm not so familiar so far with pull requests on GitHub (I'm not a professional developer and never did it before) but of course if you want I can try to upload my modified files so other members can also get it if needed.
Thank you gain for your work!
